pkg/authentication/handlers.go

```diff
@@ -253,6 +254,26 @@ func (a *API) handleLogout(w http.ResponseWriter, r *http.Request) {
 	a.cookieManager.ClearAccessTokenCookie(w)
 	a.cookieManager.ClearRefreshTokenCookie(w)
 
+	session, err := a.identities.GetIdentitySession(ctx, r.Cookies())
+	if err != nil {
+		a.logger.Errorf("failed to retrieve kratos session, err: %v", err)
+		a.badRequest(w, err)
+		return
+	}
+	if session == nil {
+		a.badRequest(w, fmt.Errorf("failed to get session id"))
+		return
+	}
+
+	sessionID := session.GetSession().Id
+	_, err = a.identities.DisableSession(ctx, sessionID)
+
+	if err != nil {
+		a.logger.Errorf("failed to disable kratos session, err: %v", err)
+		a.badRequest(w, err)
+		return
```

Comment:
nitpick: it would possibly be better to have the same error returned to the user in the logs, so
```suggestion
                err = fmt.Errorf("failed to disable kratos session, err: %v", err)
		a.logger.Error(err.Error())
		a.badRequest(w, err)
		return
```